### ISSUE TITLE
Litematica not saving

### ISSUE THREAD

**Issue by Sorenskakkongen** [maruohon/litematica#290]
Hi! So, I wanted to use litematica to copy something, but there's just one issue
When I use "Save Schematic", after having made the box, it just straight up does not save it, and I don't know what to do. If anyone could help me, that'd be lovely! Also, I am using the latest versions of malilb, litematica and fabric
![image](https://user-images.githubusercontent.com/78016570/105821374-0efe4500-5fbb-11eb-87d8-e0cfd3422808.png)

**Comment by maruohon** [comment]
If the selection goes outside of you client's loaded area, then you will be left with a schematic saving task, the status of which you can see at the bottom right of your screen when you close the menus. In that case you need to move around enough to load all the missing chunks, and then the task will complete and save the schematic to a file.

**Comment by whitefalcon95** [comment]
> If the selection goes outside of you client's loaded area, then you will be left with a schematic saving task, the status of which you can see at the bottom right of your screen when you close the menus. In that case you need to move around enough to load all the missing chunks, and then the task will complete and save the schematic to a file.

Thank you sir! I had to registrate on the website to say thank you, you solved my problem. :)

**Comment by TadaClip** [comment]
> If the selection goes outside of you client's loaded area, then you will be left with a schematic saving task, the status of which you can see at the bottom right of your screen when you close the menus. In that case you need to move around enough to load all the missing chunks, and then the task will complete and save the schematic to a file.

@maruohon its possible to save the shcem without saving the remaining chunks?

**Comment by maruohon** [comment]
@TadaClip No, at least not in the current version. If you stop/cancel the task, then it won't save anything. When/how would this even be useful, if you end up with an incomplete schematic?

**Comment by itsthecatdev** [comment]
I'm getting this issue, but theres no missing chunks ui

**Comment by WeirdToaster** [comment]
Has the status of incomplete saving changed on later versions or do i still need to load all of the chunks first?

Currently there are over 95,000 chunks queued, most of which are unnecessary for what im doing

**Comment by maruohon** [comment]
@WeirdToaster It hasn't changed.

Why do you have tons of stuff selected that you don't actually want to save? Are you using just one massive selection box that only has interesting stuff in some limited places? Then you should be using the "Normal"/Multi-Region selection mode and only select the things you actually want to save. See the [wiki article](https://github.com/maruohon/litematica/wiki/Multi%E2%80%90Region-Area-Selections-(pre%E2%80%90mod%E2%80%90rewrite)) if you are unfamiliar with how that mode works.
